tests/test_coils.py

```diff
@@ -982,8 +1002,8 @@ def test_load_makegrid_coils_header_asserts(tmpdir_factory):
 @pytest.mark.unit
 def test_repr():
     """Test string representation of Coil objects."""
-    coil = FourierRZCoil()
-    assert "FourierRZCoil" in str(coil)
+    coil = FourierPlanarCoil()
+    assert "FourierPlanarCoil" in str(coil)
```

Comment:
Any particular reason for this change? It's fine, I'm just curious.

Reply:
It avoids a self-intersection later in the test when it forms a coilset